src/main/interface/msp.c

```diff
@@ -1277,8 +1274,38 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU16(dst, currentPidProfile->itermThrottleThreshold);
         sbufWriteU16(dst, currentPidProfile->itermAcceleratorGain);
         sbufWriteU16(dst, currentPidProfile->dtermSetpointWeight);
-        break;
+        sbufWriteU8(dst, currentPidProfile->iterm_rotation);
+#if defined(USE_SMART_FEEDFORWARD)
+        sbufWriteU8(dst, currentPidProfile->smart_feedforward);
+#else
+        sbufWriteU8(dst, 0);
+#endif
+#if defined(USE_ITERM_RELAX)
+        sbufWriteU8(dst, currentPidProfile->iterm_relax);
+#else
+        sbufWriteU8(dst, 0);
+#endif
+#if defined(USE_ABSOLUTE_CONTROL)
+        sbufWriteU8(dst, currentPidProfile->abs_control_gain);
+        sbufWriteU8(dst, currentPidProfile->abs_control_limit);
+        sbufWriteU8(dst, currentPidProfile->abs_control_error_limit);
+#else
+        sbufWriteU8(dst, 0);
+        sbufWriteU8(dst, 0);
+        sbufWriteU8(dst, 0);
+#endif
+#if defined(USE_ACRO_TRAINER)
+        sbufWriteU8(dst, currentPidProfile->acro_trainer_angle_limit);
+        sbufWriteU16(dst, currentPidProfile->acro_trainer_lookahead_ms);
+        sbufWriteU8(dst, currentPidProfile->acro_trainer_gain);
```

Comment:
I don't think we need to add `acro_trainer_lookahead_ms` and `acro_trainer_gain`.  These should remain CLI-only and would very rarely need to be adjusted.  A case might even be made that they be removed and the defaults just hardcoded.  I'm just not sure we have enough use of the feature yet to confirm this.

`acro_trainer_angle_limit` should be part of MSP and exposed in the configurator.  The activation component is a box mode so it's already available in the configurator.